dump/send_pldm_cmd.cpp

```diff
@@ -18,7 +20,7 @@ void sendNewDumpCmd(uint32_t dumpId, DumpType dumpType, uint64_t dumpSize)
     uint32_t pldmDumpType = 0;
     std::stringstream ss;
     ss << std::hex << std::uppercase << dumpId;
-    std::string dumpIdString = ss.str();
+    std::string dumpIdString = std::format("{:0>8}", ss.str());
```

Comment:
std::string dumpIdString = std::format("{:0>8**X**}", dumpId); 

Adding X at the end will format with leading zeros added to ensure a total width of 8 characters, and uses uppercase for the hexadecimal digits. So you can get rid of these 2 lines 
```
std::stringstream ss;
    ss << std::hex << std::uppercase << dumpId;
```

Reply:
/xyz/openbmc_project/dump/system/entry/00000001

probably the expectation here is to transfor 1 into 00000001 so format does that